libs/portals/my-pages/health/src/screens/Vaccinations/VaccinationsWrapper.tsx

```
@@ -59,6 +62,14 @@ export const VaccinationsWrapper = () => {
           variant="utility"
           text={formatMessage(m.makeVaccinationAppointment)}
         />,
+        <Button
+          icon="informationCircle"
+          variant="utility"
+          iconType="outline"
+          onClick={() => setIsStatusModalOpen(true)}
+        >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)